File: javascript/5_kyu/simple_pig_latin.js
// https://www.codewars.com/kata/520b9d2ad5c005041100000f/train/javascript

function pigIt(str) {
  const punctuations = ["!", "?", ".", ";", ":", ","];
  return str
    .split(" ")
    .map((w) => {
      if (punctuations.includes(w)) return w;
      return w.concat(`${w[0]}ay`).substring(1);
    })
    .join(" ");
}

console.log(pigIt("Pig latin is cool !"));

File: python/5_kyu/simple_pig_latin.py
# https://www.codewars.com/kata/520b9d2ad5c005041100000f

from string import punctuation as p

def pig_it(text):
    output = []
    for word in text.split(" "):
        output.append(word if word[0] in p else f"{word[1:]}{word[0]}ay")
    return " ".join(output)
